src/document-ready.js

```diff
@@ -24,19 +24,37 @@ export function isDocumentReady(doc) {
   return doc.readyState != 'loading';
 }
 
+/**
+ * Whether the document has loaded all the css and sub-resources.
+ * @param {!Document} doc
+ * @return {boolean}
+ */
+function isDocumentComplete(doc) {
```

Comment:
Should this be called `Loaded` instead of `Complete` since that is much clearer?